### ISSUE TITLE
State and entity names displaying in lowercase with underscores 

### ISSUE THREAD

**Issue by evy0311** [home-assistant/frontend#16272]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Since a recent update in the past week (unsure of which one), my entities (only some) and their states are showing in all lower case with an underscore instead of a space. I posted on Reddit to see if others have had this issue and I am not alone. 

### Describe the behavior you expected

Entities states show like “Disarmed” or “Not Home” instead of “disarmed” or “not_home”

### Steps to reproduce the issue

1. Open HA on any browser or device
2. See entity states incorrectly named

### What version of Home Assistant Core has the issue?

2023.4.6

### In which browser are you experiencing the issue with?

All browsers/mobile apps

### Which operating system are you using to run this browser?

iOS and windows 11

**Comment by evy0311** [comment]
Not sure what you're asking for, as there isn't any code to demonstrate it. It happens on all components across all dashboards, using the built in components from HA.

**Comment by ildar170975** [comment]
> Not sure what you're asking for, as there isn't any code to demonstrate it

Every card has a code.
Since you have not posted a code - people may only guess what cards do you use, so they will not be able reproduce your case -> your issue will not be solved.

**Comment by evy0311** [comment]
> > Not sure what you're asking for, as there isn't any code to demonstrate it
> 
> 
> 
> Every card has a code.
> 
> Since you have not posted a code - people may only guess what cards do you use, so they will not be able to reproduce your case -> your issue will not be solved.


After looking, it appears just the mushroom cards are affected. Example:

```

type: custom:mushroom-alarm-control-panel-card
entity: alarm_control_panel.home
states:
  - armed_home
  - armed_away
name: Alarm
fill_container: false
show_keypad: true

```

Shows "disarmed" as seen in the screenshot above but should show "Disarmed" as it previously did.

**Comment by ildar170975** [comment]
mushroom cards are CUSTOM.
All issues must be posted in a corresponding repository.
Please close the issue.

**Comment by evy0311** [comment]
Yes, originally it was all cards displaying the issue. Which is why I opened it up in this repository. 

After clearing cache on the corresponding browsers, the default cards (built in to HASS) were fixed, with just the mushroom cards presenting the issue still.
